apps/api/index.d.ts

```diff
@@ -3,4 +3,4 @@ import * as Schema from './src/database/schema';
 
 export { Appeals, Schema };
 
-export type CaseType = 'application' | 'document';
+export type CaseType = 'appeals' | 'document';
```

Comment:
Did you meant to change this @fabfab-kc ?

Reply:
nope, well spotted.

Reply:
merging this with main will be fun...